ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -261,26 +263,35 @@ instance CardanoHardForkConstraints c => CanHardFork (CardanoEras c) where
           PCons translateLedgerStateByronToShelleyWrapper
         $ PCons translateLedgerStateShelleyToAllegraWrapper
         $ PCons translateLedgerStateAllegraToMaryWrapper
+        $ PCons translateLedgerStateMaryToAlonzoWrapper
         $ PNil
     , translateChainDepState =
           PCons translateChainDepStateByronToShelleyWrapper
         $ PCons translateChainDepStateAcrossShelley
         $ PCons translateChainDepStateAcrossShelley
+        $ PCons translateChainDepStateAcrossShelley
         $ PNil
     , translateLedgerView    =
           PCons translateLedgerViewByronToShelleyWrapper
         $ PCons translateLedgerViewAcrossShelley
         $ PCons translateLedgerViewAcrossShelley
+        $ PCons translateLedgerViewAcrossShelley
         $ PNil
     }
   hardForkChainSel =
         -- Byron <-> Shelley, ...
-        TCons (CompareBlockNo :* CompareBlockNo :* CompareBlockNo :* Nil)
+        TCons (   CompareBlockNo
+               :* CompareBlockNo
+               :* CompareBlockNo
+               :* CompareBlockNo
+               :* Nil)
         -- Shelley <-> Allegra, ...
-      $ TCons (SelectSameProtocol :* SelectSameProtocol :* Nil)
+      $ TCons (SelectSameProtocol :* SelectSameProtocol :* SelectSameProtocol :* Nil)
         -- Allegra <-> Mary, ...
-      $ TCons (SelectSameProtocol :* Nil)
+      $ TCons (SelectSameProtocol :* SelectSameProtocol :* Nil)
         -- Mary <-> ...
```

Comment:
I would maintain the same comment format
```suggestion
        -- Mary <-> Alonzo, ...
```

Reply:
Fixed!